modin/pandas/concat.py

```diff
@@ -132,3 +139,25 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: list, axis):
+    axis_getter = (lambda df: df.columns) if axis else (lambda df: df.index)
+    name_getter = (
+        (lambda df: axis_getter(df).names)
+        if isinstance(axis_getter(objs[0]), pandas.MultiIndex)
```

Comment:
Why is this `objs[0]`? Can you place a comment for why this check is necessary in this way (why not `df`)?
```suggestion
        if isinstance(objs[0].axes[axis], pandas.MultiIndex)
```

Reply:
this is a useless check, removed it